src/RPCServer.ts

```diff
@@ -44,28 +44,23 @@ const cleanupReason = Symbol('CleanupReason');
  * Events:
  * - error
  */
-interface RPCServer extends createDestroy.CreateDestroy {}
-/**
- * You must provide an error handler `addEventListener('error')`.
- * Otherwise, errors will just be ignored.
- *
- * Events:
- * - {@link events.EventRPCServerDestroy}
- * - {@link events.EventRPCServerDestroyed}
- */
-@createDestroy.CreateDestroy({
-  eventDestroy: events.EventRPCServerDestroy,
-  eventDestroyed: events.EventRPCServerDestroyed,
+interface RPCServer extends startStop.StartStop {}
+
+@startStop.StartStop({
+  eventStart: events.EventRPCServerStart,
+  eventStarted: events.EventRPCServerStarted,
+  eventStop: events.EventRPCServerStopping,
+  eventStopped: events.EventRPCServerStopped,
 })
 class RPCServer extends EventTarget {
   /**
-   * Creates RPC server.
+   * Starts RPC server.
 
    * @param obj
    * @param obj.manifest - Server manifest used to define the rpc method
    * handlers.
    * @param obj.middlewareFactory - Middleware used to process the rpc messages.
-   * The middlewareFactory needs to be a function that creates a pair of
+   * The middlewareFactory needs to be a function that starts a pair of
    * transform streams that convert `Uint8Array` to `JSONRPCRequest` on the forward
```

Comment:
Did you find and replace this? it's still creating transforms streams, the change doesn't make sense.